kolibri/plugins/management/assets/src/vue/user-create-modal.vue

```diff
@@ -2,12 +2,24 @@
 
   <div class="user-creation-modal">
     <modal btntext="Add New">
-      <div class="title" slot="header">User creation</div>
+      <div class="title" slot="header">Add New Account</div>
       <div slot="body">
-        <br>Username: <input type="text" v-model="username" placeholder="Please type in your username."><br>
-        <br>Password: <input type="text" v-model="password" placeholder="Please type in your password."><br>
-        <br>First name: <input type="text" v-model="firstName" placeholder="Please type in your first name."><br>
-        <br>Last name: <input type="text" v-model="lastName" placeholder="Please type in your last name."><br>
+        <div class="name">
```

Comment:
In the future, we might be able to clean this up and make a little more data-driven, but this is good. Thanks for taking my advice. Others can voice their concerns later lol.